tests/lib/src/reshape_utils.h

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <inference_engine.hpp>
+
+using namespace InferenceEngine;
+
+
+/**
+ * @brief Split input string using specified delimiter.
+          Return vector with input tensor information
+ */
+std::vector<std::string> split(const std::string& s, char delim);
```

Comment:
not sure `split` should be available outside the `reshape_utils.cpp`, so may be removed from here